dask_sql/physical/rel/custom/create_experiment.py

```diff
@@ -30,9 +30,6 @@ class CreateExperimentPlugin(BaseRelPlugin):
     * model_class: Full path to the class of the model which has to be tuned.
       Any model class with sklearn interface is valid, but might or
       might not work well with Dask dataframes.
-      Have a look into the
-      [dask-ml documentation](https://ml.dask.org/index.html)
-      for more information on which models work best.
       You might need to install necessary packages to use
       the models.
     * experiment_class : Full path of the Hyperparameter tuner
```

Comment:
Should we also remove mention of dask-ml from this argument description?

Reply:
Thanks! Yeah, I can remove this now.